pkgs/development/qtcreator/default.nix

```diff
@@ -54,6 +44,10 @@ stdenv.mkDerivation rec {
     Type=Application
     Categories=Qt;Development;IDE;
     __EOF__
+    # Wrap the qtcreator binary
+    wrapProgram $out/bin/qtcreator \
```

Comment:
I just pushed the updated `setupHook` to master. Now you can wrap any Qt Creator (or any other Qt program) with:
```
wrapProgram $out/bin/qtcreator \
    --prefix QT_PLUGIN_PATH : "$QT_PLUGIN_PATH" \
    --prefix QML_IMPORT_PATH : "$QML_IMPORT_PATH" \
    --prefix QML2_IMPORT_PATH : "$QML2_IMPORT_PATH"
```
During the build, those variables will be populated with the appropriate values based on the dependencies of the program.